### ISSUE TITLE
[data table] keyboard usage

### ISSUE THREAD

**Issue by paulgirard** [gephi/gephi-lite#211]
For comfort and accessibility, it should be possible to use keyboard to navigate/edit the data table.

**Comment by sim51** [comment]
Duplicate of #229
